doc/adr/0007-bundesident-url-scheme.md

```diff
@@ -0,0 +1,29 @@
+# 7. Switch from eid:// to bundesident:// scheme
+
+Date: 2022-12-13
+
+## Status
+
+Accepted
+
+## Context
+
+The Technical Guideline TR-03124-1 eID-Client – Part 1 (version 1.4 from 8. October 2021, Chapter 2.2 Full eID-Client) specifies the URL scheme for the ClientURL. 
```

Comment:
Please link the TR, see e.g. [here](https://github.com/digitalservicebund/useid-backend-service/blob/main/doc/adr/0013-format-of-eid-client-url.md).